test/tests/functional/pbs_sched_attr_updates.py

```diff
@@ -81,9 +81,9 @@ def test_basic_throttling(self):
         self.server.log_match("Type 96 request received", existence=False,
                               starttime=t, max_attempts=5)
 
-        # Sleep for 60 seconds
-        self.logger.info("Going to sleep for 60 seconds")
-        time.sleep(60)
+        # Sleep for 45 seconds
+        self.logger.info("Going to sleep for 45 seconds")
```

Comment:
Remove the comment, it is not needed with the log message.  Also, I really do want you to explain why you are sleeping.